setup/default.xml

```diff
@@ -39,6 +39,8 @@
     <entry key='commands.queueing'>true</entry>
 
     <entry key='database.ignoreUnknown'>true</entry>
+    <entry key='database.registerUnknown.regex'>\w{3,15}</entry>
```

Comment:
Can you please move this default into the key definition instead.

Reply:
Do you mean move key to traccar.xml?

Reply:
No, I mean more it into the code.

Reply:
I'm sorry, Anton I cannot catch your point. 🙏
Can you give an example to make this is clear?